app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordWithPrescribedDrugs.kt

```diff
@@ -21,6 +22,7 @@ data class TeleconsultRecordWithPrescribedDrugs(
   @Dao
   interface RoomDao {
 
+    @Transaction
```

Comment:
Is there a reason we are executing a select statement in a transaction? It normally is useful only when we are inserting data into multiple tables and we want it to be atomic.

One place where it might make sense is if we are simultaneously querying and writing to the same record, but that doesn't seem to be the case here.

Reply:
It's defined in the Room relationships docs, it also complains as warning every time the annotation processor runs.